content/en/agent/kubernetes/tag.md

```diff
@@ -19,11 +19,59 @@ The Agent can create and assign tags to all metrics, traces, and logs emitted by
+| env                         | low          | pod label `tags.datadoghq.com/env` or container envvar `DD_ENV`         | pod label or container envvar must exist            |
+| version                     | low          | pod label `tags.datadoghq.com/version` or container envvar `DD_VERSION` | pod label or container envvar must exist            |
+| service                     | low          | pod label `tags.datadoghq.com/service` or container envvar `DD_SERVICE` | `pod label or container envvar must exist           |
+| pod_phase                   | low          | pod status                                                              | n/a                                                 |
+| oshift_deployment_config    | low          | pod annotation `openshift.io/deployment-config.name`                    | openshift environment and pod annotation must exist |
+| kube_ownerref_kind          | low          | pod ownerref                                                            | pod must have an owner                              |
+| kube_deployment             | low          | pod ownerref                                                            | pod must be attached to a deployment                |
+| kube_replication_controller | low          | pod ownerref                                                            | pod must be attached to a replication controller    |
+| kube_stateful_set           | low          | pod ownerref                                                            | pod must be attached to a statefulset               |
+| persistentvolumeclaim       | low          | pod spec                                                                | a pvc must be attached to the pod                   |
+| kube_cronjob                | low          | pod ownerref                                                            | pod must be attached to a cronjob                   |
+| image_name                  | low          | pod spec                                                                | n/a                                                 |
+| short_image                 | low          | pod spec                                                                | n/a                                                 |
+| image_tag                   | low          | pod spec                                                                | n/a                                                 |
+
+### Host tag
+
+The agent can also attach kubernetes environement information as "host tags".
+
+| Tag               | Cardinality | Source                                      | Requirement                          |
+|-------------------|-------------|---------------------------------------------|--------------------------------------|
+| kube_cluster_name | low         | `DD_CLUSTER_NAME` envvar                    | `DD_CLUSTER_NAME` envvar must be set |
```

Comment:
it's not exclusively set by `DD_CLUSTER_NAME`, it can be auto-discovered using cloud provider apis